dev-libs/libstrophe/libstrophe-0.12.0.ebuild

```diff
@@ -6,7 +6,7 @@ DESCRIPTION="A simple, lightweight C library for writing XMPP clients"
 HOMEPAGE="http://strophe.im/libstrophe/"
 SRC_URI="https://github.com/strophe/${PN}/releases/download/${PV}/${P}.tar.xz"
 LICENSE="|| ( MIT GPL-3 )"
-SLOT="0"
+SLOT="0/0"
```

Comment:
This is wrong but maybe not in the way expected -- SLOT="0" has an implicit subslot of "0" already.

So, 0/0 is exactly the same as 0. What's the SONAME of the new libstrophe?

It's okay for subslot to not be equal to the SONAME, just relate to it somehow (could do 0.1 as a one off, then 1 in future or something, etc.)

Also, please cleanup old in a nseparate commit.

Reply:
> So, 0/0 is exactly the same as 0. What's the SONAME of the new libstrophe?

The new libstrophe has SONAME version 0. That's why I set the subslot 0.
But I haven't had in mind that subslot 0 is the default in this case.

> Also, please cleanup old in a nseparate commit.

Will do.

Reply:
Resolved:

* 0.12.0 now uses subslot `0.1` as suggested
* first, 0.12.0 is added in a commit, then 0.11.0 is removed in a separate commit